templates/projects/view.tmpl

```diff
@@ -70,8 +68,8 @@
 			{{end}}
 		</div>
 		<div class="divider"></div>
-	</div>
```

Comment:
Dedent above?

Reply:
To avoid unnecessary diff and unnecessary merge conflicts

Reply:
Diff is not an issue, with "hide whitespace", the push that does it will show as "whitespace-only change" to confirm.

Avoiding merge conflict is a weak argument imho to not do a cleanup.

Reply:
It's not a weak reason now since there are a lot of frontend refactoring.

Otherwise some PRs would lose changes when resolving conflicts, for example, I fixed merge errors in #25625 eae7940f6c02bb0aa274d19b32a24972c806f040  b0bd1ea660d7ae394253fb0be18561ccb4ffe3b0  and 6e3c78240217c8f14e9ed1b98d98993d32e96a0f.

If there is no "indent" change, git can merge most of the changes flawlessly.

Reply:
Well if you persist, I won't push further.

Reply:
Thank you for understanding. I will clean them in a separate PR to avoid unnecessary conflicts.

Actually, not only this page, there are other pages.

Reply:
Reformat some templates #25756